src/app/components/LinkedData/__snapshots__/index.test.tsx.snap

```diff
@@ -8,12 +8,6 @@ exports[`LinkedData bylineLinkedData should change author contents and locationC
       "@type": "Article",
       "author": {
         "@type": "Person",
-        "image": "https://ichef.bbci.co.uk/images/ic/1024x576/p063j1dv.jpg",
-        "name": "John",
-        "sameAs": [
-          "https://www.bbc.co.uk/news/topics/cg2gmrxlde0t",
-          "https://twitter.com/BBCNews",
-        ],
```

Comment:
Did we expect this to have changed?

Reply:
no! this is because the bylineLinkedData mock data defined in the test file still had it as an object, instead of an object inside an array - fixed now !